text2text/identifier.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import pickle
+import text2text as t2t
+
+
+class Identifier(t2t.Vectorizer):
+
+    def _get_model():
+        """Download and loads model from google drive"""
+
+        file_id = "1pB1GkWiafx23jmND6psg1cUyBqRq0rVY"
+        model_file = "identifier.pkl"
+        URL = "https://docs.google.com/uc?export=download"
+        session = requests.Session()
+        response = session.get(URL, params = { 'id' : file_id }, stream = True)
+        for key, value in response.cookies.items():
+            if key.startswith('download_warning'):
+                params = { 'id' : file_id, 'confirm' : value }
+                response = session.get(URL, params = params, stream = True)
+
+        CHUNK_SIZE = 32768
+        with open(model_file, "wb") as f:
+            for chunk in response.iter_content(CHUNK_SIZE):
+                if chunk: # filter out keep-alive new chunks
+                    f.write(chunk)
+        
+        model = pickle.load(open("identifier.pkl", 'rb'))
```

Comment:
When I tested the model, it returned "hy" for "hello" and "ja" for "你好!". Is this consistent with your testing as well?

Reply:
Yeah, there is a big problem of not identifying short texts correctly for some languages. I'm guessing it's because most of the training data are long sequences.

Reply:
We can train the model on shorter texts and update it over time with improvements. For now, let's publish what we have and note in the README known limitations